### PR TITLE
STRMFS-16 | D | Migrations for the basic content_information database structure 

### PR DESCRIPTION
**Pull request by streamTinder**

### CODE REVIEW

File: database/migrations/2021_01_21_173609_create_rating_services_table.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateRatingServicesTable extends Migration
+{
+    const STR_LEN = 256;
```
Comment (lplatzer) [inline database/migrations/2021_01_21_173609_create_rating_services_table.php:9]: Add 
`const URL_LEN = 2083;`

File: database/migrations/2021_01_21_173609_create_rating_services_table.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    public function up()
+    {
+        Schema::create('rating_services', function (Blueprint $table) {
+            $table->id();
+            $table->string('service_name', self::STR_LEN);
+            $table->longText('rating_description');
+            $table->integer('max_rating');
+            $table->integer('min_rating');
+            $table->float('step_size');
+        });
```
Comment (lplatzer) [inline database/migrations/2021_01_21_173609_create_rating_services_table.php:24]: Add table for links to rating services:
`$table->string('logo_url', self::URL_LEN);`

File: database/migrations/2021_01_21_173609_create_rating_services_table.php (hunk 1 above)
Comment (lplatzer) [inline database/migrations/2021_01_21_173609_create_rating_services_table.php:9]: Add to enable icon linking for rating services
`const URL_LEN = 2083;`

File: database/migrations/2021_01_21_173609_create_rating_services_table.php (hunk 2 above)
Comment (lplatzer) [inline database/migrations/2021_01_21_173609_create_rating_services_table.php:24]: Add table for links to rating services:
`$table->string('logo_url', self::URL_LEN);`

**Review by lplatzer — CHANGES_REQUESTED** [review]

File: database/migrations/2021_01_21_173609_create_rating_services_table.php (hunk 1 above)
Comment (streamTinder) [reply database/migrations/2021_01_21_173609_create_rating_services_table.php:10]: Did that

File: database/migrations/2021_01_21_173609_create_rating_services_table.php (hunk 2 above)
Comment (streamTinder) [reply database/migrations/2021_01_21_173609_create_rating_services_table.php:26]: Did that

**Review by lplatzer — APPROVED** [review]

**Pull request merged**